bullet/src/main/resources/org/bytedeco/bullet/include/clew_stubs.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2022 Andrey Krainyak
+ *
+ * Licensed either under the Apache License, Version 2.0, or (at your option)
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation (subject to the "Classpath" exception),
+ * either version 2, or any later version (collectively, the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.gnu.org/licenses/
+ *     http://www.gnu.org/software/classpath/license.html
+ *
+ * or as provided in the LICENSE.txt file that accompanied this code.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+
+// This #ifdef disables these stubs during native code compilation,
+// as the stubs should be used for generation of the Java-side
+// code only, and the native code should use the original definitions
+// provided by clew's header.
+#ifdef XXXXXXXXXX
```

Comment:
Are you saying JavaCPP isn't able to parse that header file for clew? What's wrong with the output? We should try to fix that first before adding some custom script. It looks like it's basically just OpenCL though. We can't link with the presets for OpenCL and use those functions from there?

Reply:
This clew stuff also doesn't seem to work on Mac and 32-bit Windows. What's the reason we can't use standard OpenCL?

Reply:
Sure, let's try to fix it :)

The reason why I did this is that each of those function names is actually a [macro definition](https://github.com/bulletphysics/bullet3/blob/22d9eec148d3a7dec9d7b10a4344808bf766f73a/src/clew/clew.h#L2628) that is substituted by a [global variable](https://github.com/bulletphysics/bullet3/blob/22d9eec148d3a7dec9d7b10a4344808bf766f73a/src/clew/clew.h#L2526) of a [function type](https://github.com/bulletphysics/bullet3/blob/22d9eec148d3a7dec9d7b10a4344808bf766f73a/src/clew/clew.h#L1972). Sorry, I just couldn't figure out a better way.

Reply:
Ok, but why do we need any of this? What's missing from the presets for OpenCL? OpenCV is another library that uses OpenCL but doesn't need to link with any external version of OpenCL. It works just fine with the presets for OpenCL at runtime though.

Reply:
> This clew stuff also doesn't seem to work on Mac and 32-bit Windows. What's the reason we can't use standard OpenCL?

Yeah, initially I thought about plugin in the javacpp's OpenCL, but than I found, that `clew` support is [hardcoded](https://github.com/bulletphysics/bullet3/blob/22d9eec148d3a7dec9d7b10a4344808bf766f73a/src/Bullet3OpenCL/CMakeLists.txt#L3) into the build script, and `clew`'s source code itself is [included](https://github.com/bulletphysics/bullet3/tree/master/src/clew) into the source code of `bullet`. So, I thought that maybe there are some special reasons behind that, and finally decided to just map `clew.h`.

I think, it might be possible to override it by patching the build script and maybe a bunch of cpp files. I haven't tried it though. Should I go that way?

Reply:
Actually, I didn't find a working OpenCL example. They aren't even included into the [cmake build](https://github.com/bulletphysics/bullet3/blob/22d9eec148d3a7dec9d7b10a4344808bf766f73a/examples/ExampleBrowser/CMakeLists.txt#L151). I tried to add them manually, but there were compilation errors in the example's source code. The [comments](https://github.com/bulletphysics/bullet3/blob/22d9eec148d3a7dec9d7b10a4344808bf766f73a/examples/ExampleBrowser/OpenGLExampleBrowser.cpp#L855) in the ExampleBrowser say that it is still experimental.

Reply:
There is also [this](https://github.com/bulletphysics/bullet3/issues/1401). And the git log says that there were only 6 commits made into Bullet3OpenCL since that time. And less than 20 commits into any of the `Bullet3` folders. Maybe we should just drop it.

Reply:
> Ok, but why do we need any of this? What's missing from the presets for OpenCL? OpenCV is another library that uses OpenCL but doesn't need to link with any external version of OpenCL. It works just fine with the presets for OpenCL at runtime though.

Yeah, I don't see any reason why it shouldn't work with the standard OpenCL. As I understand, `clew` is just an API loader for OpenCL - like `glew` for OpenGL. I expect the API would be identical to the standard one. As I said, if we want to switch to the standard OpenCL, we are gonna need to patch some files.

Reply:
I will have a look at the 32-bit Windows build later.

Reply:
I don't mean to disable Bullet3OpenCL or to somehow make it link with another version of OpenCL, I just mean to remove the mappings for clew. We should be able to pass objects around with any other version of OpenCL, so it should be fine. I've pushed that in the latest commit. The only issue is that we get opaque Pointer for OpenCL objects, but that's not a big issue. Let me know if that looks alright.

Reply:
Looks good to me :) Thank you for helping out with this!